modules/Sources/Features/AddressDetails/AddressDetailsView.swift

```diff
@@ -36,11 +36,11 @@ public struct AddressDetailsView: View {
                 }
             }
             .applyScreenBackground()
-            .zashiBack()
-            .zashiTitle {
-                Text(L10n.AddressDetails.receiveTitle.uppercased())
-                    .font(.custom(FontFamily.Archivo.bold.name, size: 14))
-            }
+//            .zashiBack()
+//            .zashiTitle {
+//                Text(L10n.AddressDetails.receiveTitle.uppercased())
+//                    .font(.custom(FontFamily.Archivo.bold.name, size: 14))
+//            }
```

Comment:
Should we rather remove it than comment out?

Reply:
Removed